.github/workflows/main.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10', 'pypy3']
+        python-version: ['3.7', '3.8', '3.9', '3.10', 'pypy3']
```

Comment:
With this version of python for GitHub Actions, `pypy3` will run in 3.6 mode.

I assume this is not intended, as you specifically drop 3.6 here.

I recently updated this for Onyo: https://github.com/psyinfra/onyo/pull/144/commits/3b29a717597a0b4af9c00d838218caea590574aa